docker-compose.yaml

```diff
@@ -83,6 +83,7 @@ services:
       - MAX_DISCOVER_WORKERS=${MAX_DISCOVER_WORKERS}
       - MAX_SPEC_WORKERS=${MAX_SPEC_WORKERS}
       - MAX_SYNC_WORKERS=${MAX_SYNC_WORKERS}
+      - MAX_NOTIFY_WORKERS=${MAX_NOTIFY_WORKERS}
```

Comment:
Should we add the enabled here as well?

Reply:
this? SHOULD_RUN_NOTIFY_WORKFLOWS

I'm not sure exactly how all these env variables are set/passed around - I didn't see others of this type in here so I didn't add it, but maybe it needs to be set here?

Reply:
Yes `SHOULD_RUN_NOTIFY_WORKFLOWS`, we have similar ones like `USE_STREAM_CAPABLE_STATE` or `RUN_DATABASE_MIGRATION_ON_STARTUP`

Reply:
And I forgot that we will need to add that to the kustomize files (in the `charts` folder)

Reply:
I'm not seeing this. @benmoriceau is there documentation on how to add an env variable so it is added in all the right places?